### PR TITLE
Log the contained level id as the attempted level id

### PR DESCRIPTION
**Pull request by balderdash**
Milestone posts from contained level completions use the level id of the contained level. At the moment, the api_controller is still logging the attempts under the level id of the outer level, rather than contained level. This mismatch is giving us a bunch of spurious activitymonitor complaints.

This PR updates the logging of the level attempt to use the contained level id as well (when present). Logentry logs are only consumed by activitymonitor, so this shouldn't mess with anything else.

### CODE REVIEW

File: dashboard/app/controllers/api_controller.rb [hunk 1]
```diff
@@ -204,7 +204,7 @@ def user_progress_for_stage
       slog(
         tag: 'activity_start',
         script_level_id: script_level.try(:id),
-        level_id: level.id,
+        level_id: level.contained_levels.empty? ? level.id : level.contained_levels.first.id,
```
Comment (ashercodeorg) [inline dashboard/app/controllers/api_controller.rb:207]: Is there a reason we are logging only the first contained level, rather than logging all of them?

Comment (balderdash) [reply dashboard/app/controllers/api_controller.rb:207]: We don't actually render more than one contained level anywhere. There's one level that has multiple contained levels, but it only shows the first and isn't actually used in a script: https://levelbuilder-studio.code.org/levels/4562

Comment (ashercodeorg) [reply dashboard/app/controllers/api_controller.rb:207]: Then why do we store it as an array?

Comment (balderdash) [reply dashboard/app/controllers/api_controller.rb:207]: I'm guessing there was a plan to one day support multiple, but nobody got around to that.

**Pull request merged**